assets/src/edit-story/components/inspector/document/test/slug.js

```diff
@@ -60,4 +65,37 @@ describe('SlugPanel', () => {
     const url = getByRole('link', { name: 'https://example.com/foo' });
     expect(url).toBeDefined();
   });
+
+  it('should not display permalink', async () => {
```

Comment:
Hmm, is the description accurate here? Looks like it's testing the min-max of the permalink and not the displaying of permalink.

Reply:
Yep, fixed.